ui/src/components/home/ResourceTab.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography, TextField } from '@mui/material';
+import { FC, ReactElement } from 'react';
+import TextSnippetIcon from '@mui/icons-material/TextSnippet';
+
+const GroupTab: FC = (): ReactElement => {
+  return (
+    <Box
+      sx={{
+        padding: '16px',
+        border: '1px solid #ccc',
+        borderRadius: '4px',
+        backgroundColor: 'primary.light',
+      }}
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          margin: 'auto',
+          justifyContent: 'space-between',
+        }}
+      >
+        <Typography variant="h3" color={'primary.main'}>
+          Resources
+        </Typography>
+        <TextSnippetIcon sx={{ fontSize: '20', marginX: '1vw' }} color="primary" />
```

Comment:
why add fontSize in this icon here? if it's for the icon to be the same size as the font, you can try do this, i think it looks cleaner:
```
            <Typography variant="h4" sx={{ display: 'flex', alignItems: 'center' }} gap={1} py={1}>
              <LibraryMusicIcon color="primary" />
              Lyrics & Chords
            </Typography>
```